monkey/tests/monkey_island/conftest.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def mocked_server_configs_dir(mocked_data_dir):
```

Comment:
In this case, the directory isn't mocked. It's a real directory. Maybe `test_server_configs_dir`?